contracts/UFragmentsPolicy.sol

```diff
@@ -264,6 +264,40 @@ contract UFragmentsPolicy is Ownable {
         );
     }
 
+    /**
+     * @return Computes the percentage of supply to be added or removed.
```

Comment:
```suggestion
     * @return The percentage of supply to be added or removed.
```
Usually the @return goes last in the NatSpec block.

Reply:
Done.